raft/log.go

```diff
@@ -940,7 +1052,7 @@ func (l *Log) Wait(index uint64) error {
 		} else if appliedIndex >= index {
 			return nil
 		}
-		time.Sleep(1 * time.Millisecond)
+		time.Sleep(100 * time.Millisecond)
```

Comment:
I moved it to a constant for now. We can change it to a `var` or make it a setting on the `Log` if we need to. Eventually I'd like to take out the sleep and just use channels. (88ef994)